src/modules/interactions.js

```diff
@@ -0,0 +1,16 @@
+export default function interactions() {
+  const checkboxes = document.querySelectorAll('.checkbox');
+  checkboxes.forEach((box) => {
+    box.addEventListener('change', (e) => {
+      let toDos = [];
+      toDos = JSON.parse(localStorage.getItem('toDos') || '[]');
+
+      toDos.forEach((obj) => {
+        if (e.target.classList.contains(obj.index)) {
+          obj.completed = true;
```

Comment:
- [ ] **Optional:** The completed state is true on reload but the styles are gone. You could attempt to restore the completed styles on reload.